### DISCUSSION TITLE
In class exercise for Feb 06 (Up-Goer Five Text Editor)

### DISCUSSION THREAD

**Discussion by sr320** [course-fish521-2023/assistance#5] [Show and tell]
This is where you will post...
click the “PERMA-LINK” button and copy/paste the URL

**Comment by bzhang111** [comment]
http://splasho.com/upgoer5/?i=FFOfnJgyVUEiVTkyLKWhVTSvo3I0VUquqTIlVTShnJ1uoUZhVPOGo21yVUyyLKWmVUEbMKxtnTS2MFOgo3WyVTAbnJkxpzIhYPOuozDtp29gMKEcoJImVUEbMKxtnTS2MFOfMKAmYvOWVUquoaDtqT8tn25iqlObo3ptLJ5xVUqbrFO0nTymVTuupUOyoaZhVPOHnTymVTAuovObMJkjVT1yVTgho3ptnT93VT1uoaxtqTuypzHtoJS5VTWyVTkuqTIlYvNt

**Comment by EveMHallock** [comment]
https://splasho.com/upgoer5/?i=DJ5coJSfplO0nTS0VTMfrFOhMJIxVUEiVTuuqzHtLzSvnJImYPOvqKDto3EbMKVtLJ5coJSfplOzpz9gVTEcMzMypzIhqPOjoTSwMKZtMJS0VUEbMJ0hVSEbnKZtnKZtLFOjpz9voTIgVUEbLKDtFFO3nJkfVUA0qJE5YvOBolOiozHtnTSmVTSxMTIxVUEbMKAyVTShnJ1uoUZtqKNtLzIzo3WyYPOmolO3MFOxolOho3Dtn25iqlObo3ptoJShrFO0nTIlMFOupzHto3VtnT93VUEbMKWyVUqcoTjtLzHtLJM0MKVtqTuyVT90nTIlVTShnJ1uoUZtLKWyVTgcoTkyMP4t

**Comment by nateredon** [comment]
http://splasho.com/upgoer5/?i=FFOfnJgyVUEiVUA0qJE5VTShnJ1uoUZtqTuuqPOfnKMyVTyhVTucM2ttLJ5xVTAioTDtpTkuL2ImYvOHnTyhM3ZtLKWyVTAbLJ5anJ5aVUS1nJAeoUxtMz9lVUEbMJ0tLJ5xVUEbMJylVTkcqzImVTSlMFO2MKW5VTuupzDhVRxtLJ0tnJ50MKWyp3EyMPOcovO1ozEypaA0LJ5xnJ5aVUEbMFOjpz9voTIgplO0nTI5VTSlMFOzLJAcozptLJ5xVUElrJyhMlO0olOwnTShM2HtnT93VUqyVTuyoUNtqTuyoF4t

**Comment by cirquegammelin** [comment]
https://splasho.com/upgoer5/?i=FFOmqUIxrFOuozygLJkmVUEbLKDtoJ92MFOvMKE3MJIhVTEcMzMypzIhqPOvo2EcMKZto2Ltq2S0MKVhVRxtq2ShqPO0olO1ozEypaA0LJ5xVTuiqlO0nTImMFOuozygLJkmVTkcqzHtLJ5xVTqlo3pfVTShMPObo3ptqT8toJShLJqyVUEbMJylVT51oJWypaZtqT8toJSeMFOmqKWyVUEbMKxtq2yfoPOuoUqurKZtLzHtLKWiqJ5xYvNXDKZtqTuyVUqipzkxVTAbLJ5aMKZtLJ5xVTWyL29gMKZtq2SloJIlYPO3MFOhMJIxVUEiVTgho3ptq2uuqPO3nJkfVTuupUOyovO0olO0nTImMFOuozygLJkmYvOHnTymVTymVUMypaxtnJ1jo3W0LJ50VTWyL2S1p2HtqTuypzHtLKWyVT1uoaxtpTIipTkyVUqbolOwLKEwnPO0nTImMFOuozygLJkmVUEiVTIuqPOuozDtoJSeMFOgo25yrF4t

**Comment by kinmic** [comment]
https://splasho.com/upgoer5/?i=H29gMFOuozygLJjtoaIgLzIlplOaolO1pPOuozDtMT93ov4tIT8tp2IyVTuiqlOgqJAbVUEbMKxtM28tqKNtLJ5xVTEiq24tLaI5YPO3MFOmqUIxrFO0nTIgVTI2MKW5VUyyLKVtLKDtqTuyVUAuoJHtqTygMF4tI2Htp3E1MUxtqTuyoFOcovO0nTHtqzIlrFOmLJ1yVUqurFOyqzIlrFO5MJSlYPOuqPO0nTHtp2SgMFO0nJ1yYvOWMvO3MFOwnTShM2Htoz90nTyhMljtqTuyovOwnTShM2ImVTyhVT91pvOhqJ1vMKWmVUAbo3ptL2uuozqyplOcovO0nTHtoaIgLzIlplOiMvOuozygLJkmYvOPqKDtq2uuqPOcMvO0nTHtq2S2MKZtLJ5xVUqcozDtoJSeMFOuozygLJkmVTuupzEypvO0olOmqUIxrG8tG3IlVT51oJWypaZtoJyanUDtL2uuozqyVTWyL2S1p2Hto2Lto3IlVUA0qJE5VTuuplOwnTShM2IxYPOho3DtqTuyVTShnJ1uoUZtqTuyoKAyoUMypl4=

**Comment by n-grace-henry** [comment]
https://splasho.com/upgoer5/?i=HTIipTkyVT5yMJDtqTucozqmVUEbLKDtL29gMFOzpz9gVUEbMFO3LKEypv4tI2uyovOjMJ9joTHtL2uuozqyVUqbLKDtqTuyVUqipzkxVTymVTkcn2HfVUEbMFOzo29xVUEbLKDtLJ5coJSfplOcovO0nTHtq2S0MKVtMJS0VTSfp28tL2uuozqyplOuVTkiqPOuozDtqTucplOwLJ4tLzHtLzSxVT9lVTqio2DfVTW1qPO3MFOxo24aqPOeoz93VTuiqlO0nTI5VTSlMFOwnTShM2yhMl4tIT8tLzHtpzIuMUxtMz9lVT1ipzHtL2uuozqyplO0olO0nTHtq29loTDfVUqyVUAbo3IfMPOeoz93VTuiqlO0nTHtq29loTDtnKZtL2uuozqcozptMz9iMPOzo3Vtq2S0MKVtLJ5coJSfpl4tFFOfnJgyVUEiVTgho3ptnT93VTMio2DtnJ4tqTuyVUquqTIlVUqcoTjtL2uuozqyYvN=

**Comment by KellyAnnNeal** [comment]
https://splasho.com/upgoer5/?i=FFOmqUIxrFOuozygLJkmVTyhVUEbMFO3LKEypv4tFFO3LJ50VUEiVUIhMTIlp3EuozDtq2u5VUEbMKxtLKWyVTygpT9lqTShqPOzo3VtqTuyVUqipzkxYvOWVTSfp28tq2ShqPO0olO1ozEypaA0LJ5xVTuiqlO0nTI5VTSlMFOcoKOipaEuoaDtMz9lVT90nTIlVTShnJ1uoUZtLJ5xVT90nTIlVTkcMzHtqT8tL29hqTyhqJHto24hVRxtq2yfoPO1p2HtL29gpUI0MKWmVUEiVTAioKOfMKEyVUEbnKZtq29lnl4t

**Comment by cvaage** [comment]
https://splasho.com/upgoer5/?i=VRxtLJ0tnJ50MKWyp3EyMPOcovO3LKEypvOuozygLJkmVUEbLKDtp2uiqJkxVT5iqPOvMFO0nTIlMF4tFFO1p2HtL29gpUI0MKWmVUEiVUAyMFObo3ptqTuyrFO3nJkfVTWyVUqcoTjtLzHtMaWcMJ5xplOipvOho3DhVSqyVTEiVT5iqPOeoz93VTuiqlObo3DtLJylVUqcoTjtnUIlqPO0nTHtLJ5coJSfplO3MFOfnJgyVT9lVTAbLJ5aMFObo3ptqTuyVUE3olOapz91pUZtq29lnlO0o2qyqTuypv4t

**Comment by KalenaKdB** [comment]
https://splasho.com/upgoer5/?i=FFOuoFOfo29enJ5aVTS0VTEyLJDtLJ5coJSfplOzpz9gVTRtoT9hMlO0nJ1yVTSaolO0olOmMJHtq2uuqPOenJ5xVT9zVTMio2DtqTuyrFO1p2IxVUEiVTIuqPjtq2uypzHtqTuyrFO1p2IxVUEiVTkcqzHfVTShMPO3nUxtqTuyrFOxo24aqPOfnKMyVUEbMKWyVTShrJ1ipzHhVRxtq2yfoPO1p2Htq2uuqPOWVTkyLKWhVUEiVUqlnKEyVTWyqUEypvOjoTShplO0olOgLJgyVUA1pzHtqTuyrFOxo24aqPOaolOuq2S5VTWyL2S1p2HtpTIipTkyVT5yMJDtqTuyoFOzo3VtMz9iMP4tDJkmoljtqTuyp2HtLJ5coJSfplO0nTS0VTkcqzHtnJ4tq2S0MKVtLKWyVUMypaxtnJ1jo3W0LJ50VUqbMKWyVUEbMKxtoTy2MF4t

**Comment by valeste** [comment]
https://splasho.com/upgoer5/?i=FFOuoFOfo29enJ5aVTyhqT8tq2S5plO0olOgLJgyVTRtL2IfoPOfnJ5yVT9zVTShVTShnJ1uoPO0nTS0VTuuplOhMKMypvOvMJIhVT1uMTHtnJ50olOuVTAyoTjtoTyhMFOvMJMipzHhVRxtq2yfoPOxolO0nTymVTW5VT1un2yhMlO0nTHtLzImqPObo21yVTMipvO0nTHtL2IfoUZtMzylp3DtLJ5xVUEbMJ4tpUI0qTyhMlO0nTIgVTyhqT8tLFObLKWxVUAyqUEcozptqTuuqPO3nJkfVTMipzAyVUEbMFOapz91pPOiMvOwMJkfplO0olOwnTShM2HtqTuyoKAyoUMypl4tIT8toJSeMFO0nTHtLzImqPObo21yVTMipvO0nTHtL2IfoUZtFFO3nJkfVTAbMJAeVTEcMzMypzIhqPO3LKymVUEiVTqlo3ptqTuyoF4tFFO3nJkfVTEyL2yxMFO0nTS0VTy0VTymVUEbMFOvMKA0VUqurFOiMvOapz93nJ5aVUEbMJ0tLaxtL2uyL2gcozptnT93VTMup3DtqTuyrFOapz93VTShMPObo3ptoT9hMlO0nTI5VTkcqzHhVSEbMJ4tFFO3nJkfVT1un2HtqTuyVUAyqUEcozptMz9lVUEbMJylVTuioJHtnTSlMTIlVTW5VTAbLJ5anJ5aVUEbnJ5aplOuLz91qPObo3ptFFOeMJIjVUEbMJ0tqTuuqPOupzHtn25iq24tqT8tL2uuozqyVTqlo3IjplOiMvOuozygLJkmYvN=

**Comment by giojacuzzi** [comment]
http://splasho.com/upgoer5/?i=FFqgVTyhqTIlMKA0MJDtnJ4tqKAcozptp291ozDtqT8tp2IyVTuiqlOuozygLJkmVUWyp3OiozDtqT8tL2uuozqyplOcovO0nTIcpvOfnKMypl4tDaxtoTymqTIhnJ5aVUEiVUEbMJylVTAuoTkmYPO3MFOwLJ4toTIupz4tq2uuqPO0nTI5VTSlMFOxo2yhMlOvo3EbVTW5VUEbMJ1mMJk2MKZtLJ5xVUEiM2I0nTIlYPOuozDtnT93VUEbLKDtnKZtMTyzMzIlMJ50VTMlo20tq2uuqPO3MFO3o3IfMPOyrUOyL3DhVSEbnKZtL2ShVTuyoUNtqKZtoTIupz4tq2uuqPO3MFOhMJIxVUEiVTAbLJ5aMFO0olOgLJgyVUEbnJ5aplOvMKE0MKVtMz9lVTWiqTttLJ5coJSfplOuozDtpTIipTkyYt==

**Comment by scuba-sarah** [comment]
https://splasho.com/upgoer5/?i=FFOuoFOfo29enJ5aVTyhqT8tLJ4tLJ5coJSfVUEbLKDtnKZtoTy2nJ5aVUqbMKWyVTy0VUAbo3IfMT4aqP4tFFOuoFO0paycozptqT8tMzyhMPOiqKDtq2uuqPOuozDtq2uypzHtqTucplOuozygLJjtnKZtMJS0nJ5aYvOWVT5yMJDtqT8tn25iqlO3nTS0VTy0VTymVTIuqTyhMlOmolO3MFOwLJ4tqJ5xMKWmqTShMPOcMvO0nTymVTShnJ1uoPOcplOwLKImnJ5aVUOlo2WfMJ1mVTMipvOiqTuypvOuozygLJkmVUEbLKDtoTy2MFO0nTIlMF4=

**Comment by cskalley** [comment]
https://splasho.com/upgoer5/?i=FFOfnJgyVUEiVUA0qJE5VTShnJ1uoUZtqTuuqPOfnKMyVTyhVTAioTDtq2S0MKVfVTImpTIwnJSfoUxtqTuyVTShnJ1uoPO0nTS0VTkio2gmVTkcn2HtLFOxo29lVTShMPOfnKMyplOiovO0nTHtLz90qT9gVT9zVUEbMFO3LKEypv4tPxyhVQVjZwRfVTRtozI3VUE5pTHto2LtqTucplOxo29lYJkcn2HfVUquqTIlVTShnJ1uoPO3LKZtMz91ozDhVRxtLJ0tp3E1MUycozptqTucplOhMKptqUyjMFOiMvOuozygLJjtqT8tqTIfoPOjMJ9joTHtq2uuqPOcqPOfo29eplOfnJgyYvNt

**Comment by sarahteman** [comment]
https://splasho.com/upgoer5/?i=FFOmqUIxrFObo3ptLJ5coJSfplOlMKAjo25xVUEiVTAbLJ5aMKZtnJ4tqTuynKVtoTy2MKZhVSAioJHtLJ5coJSfplO3nJkfVT5iVTkiozqypvOfnKMyVTyzVUEbMJylVTuioJHtnKZtLzSxYvOHnTImMFOuozygLJkmVTAiqJkxVTEcMF4tI2HtnT9jMFOho3Du

**Comment by klasdin** [comment]
http://splasho.com/upgoer5/?i=GKxtMz9wqKZtoT9in3ZtLKDtnT93VTShnJ1uoUZtMJS0VUEcoaxtLzSxVUEbnJ5aplOcovO3LKEypv4tITuyp2HtLzSxVUEbnJ5aplOupzHtoTIzqPOuMaEypvObqJ1uoaZtqKAyVUEbMJ0tLJ5xVTIhMPO1pPOcovO3LKEypv4tG25wMFO0nTI5VTSlMFOyLKEyovjtqTuyp2HtqTucozqmVTAuovOvMFObLKWxVUEiVUAyMFOuozDtoJS0L2ttqT8toTSlM2IlVUEbnJ5apl4tI2Htq2ShqPO0olOmqUIxrFObo3ptoJShrFO0nJ55VUEbnJ5aplO0nTIlMFOupzHtnJ4tqTuyVUqipzkxYt==

**Comment by lizboggs** [comment]
https://splasho.com/upgoer5/?i=FFOfnJgyVUEiVTkio2ftLKDtLJ5coJSfplOcovO0nTHtq2S0MKVtLJ5xVUAyMFO3nTS0VTqlo3IjplO0nTI5VTSlMFOcovO1p2yhMlO0nTHtoTI0qTIlplOuozDtq29lMUZtqTuuqPOgLJgyVUEbMJylVTWiMTyyplO3o3WeYvOWVTAuovOuoUAiVTkio2ftLKDtqTuynKVtnT93VUEbMJylVTMuoJyfrFObLKZtL2uuozqyMPOiqzIlVUEcoJHhVSAioJHto2LtqTuyp2HtLJ5coJSfVTqlo3IjplOupzHtqT9iVUAgLJkfVTShMPO0nTS0VT1yLJ5mVUEbLKDtqTuyrFOupzHtnJ4tqUWiqJWfMF4tFFOwLJ4tqKAyVUEbnKZtp3E1MzLtqT8toJSeMFOmqKWyVUqyVTSlMJ4aqPO0LJgcozptqT9iVT1uoaxtMaWioFOapz91pUZtqTuuqPObLKMyVTkiqlOhqJ1vMKWmVTShMPOznJ5xVTuiqlO0olOvMKA0VT1uozSaMFO0nTIgVTSmVT91pvO3o3WfMPOwnTShM2ImVUAiVUEbLKDtqTuynKVtoaIgLzIlplOaMKDtLzyaM2IlYPO3nTywnPOcplOao29xVTMipvO0nTHtq2S0MKVtLJ5xVTMipvO1pl4=

**Comment by KatLeigh11** [comment]
http://splasho.com/upgoer5/?i=GJSenJ5aVUEcMKZtLzI0q2IyovOjMJ9joTHtq2uiVTkcqzHtL2kip2HtqT8tq2S0MKVh

**Reply by KatLeigh11** [reply]
really think this is pretty pointless since I can't use the words "coastal" "network" "digital" or "create". I understand the intent but this much restriction is pretty non-constructive.
